core/geometry/src/curve/CurveOps.ts

```diff
@@ -130,24 +130,24 @@ export class CurveOps {
    * @param planeTolerance tolerance for considering a closed chain to be planar. If undefined, only create Path. If defined, create Loops for closed chains within tolerance of a plane.
    * @returns chains, possibly wrapped in a [[BagOfCurves]].
    */
-  public static collectChains(fragments: AnyCurve[], gapTolerance: number = Geometry.smallMetricDistance, planeTolerance: number | undefined = Geometry.smallMetricDistance): AnyChain | undefined {
+  public static collectChains(fragments: AnyCurve[], gapTolerance: number = Geometry.smallMetricDistance, planeTolerance?: number): AnyChain | undefined {
```

Comment:
why did you remove the default value for `planeTolerance`?

Reply:
Because "undefined" has a specific behavior (return only Paths) that a default value was not allowing. (This bug is explained further in the PR description and issue #6782.)